pkg/storage/allocator_scorer.go

```diff
@@ -60,6 +60,22 @@ const (
 	// hypothetically ping pong back and forth between two nodes, making one full
 	// and then the other.
 	rebalanceToMaxFractionUsedThreshold = 0.925
+
+	// The rangeRebalanceThreshold in the options specifies a percentage based
```

Comment:
I was a bit confused reading the comment and it seems to allude to some things that aren't true in the current state. Maybe this is clearer? I might be losing some of the fidelity.

// minRangeRebalanceThreshold is the number of replicas by which a store must deviate from the mean number of replicas to be considered overfull or underfull. This absolute bound exists to account for deployments with a small number of replicas to avoid premature replica movement. With few enough replicas per node (<<30), a rangeRebalanceThreshold of 5% (the default at time of writing, see below) would otherwise result in rebalancing at one replica above/below the mean, which could lead to a condition that would always fire. Instead, we only consider a store full/empty if it's at least minRebalanceThreshold away from the mean.

Reply:
updated